NoDenotation.owner on Seq of intersection passed to java varargs

`ElimRepeated.seqToArray` assumes that the element type has a class symbol.

Code:
```scala
trait A
trait B
object O {
  def m(x: Seq[A & B]) = java.util.Arrays.asList(x: _*)
}
```

Output:
```
exception occurred while compiling RepeatedSeq.scala
java.lang.AssertionError: NoDenotation.owner while compiling RepeatedSeq.scala
Exception in thread "main" java.lang.AssertionError: NoDenotation.owner
	at dotty.tools.dotc.core.SymDenotations$NoDenotation$.owner(SymDenotations.scala:1929)
	at dotty.tools.dotc.core.SymDenotations$SymDenotation.typeRef(SymDenotations.scala:1166)
	at dotty.tools.dotc.transform.ElimRepeated.seqToArray(ElimRepeated.scala:103)
	at dotty.tools.dotc.transform.ElimRepeated.$anonfun$transformApply$2(ElimRepeated.scala:82)
	at dotty.tools.dotc.core.Decorators$ListDecorator$.zipWithConserve$extension(Decorators.scala:108)
	at dotty.tools.dotc.transform.ElimRepeated.transformApply(ElimRepeated.scala:78)
	at dotty.tools.dotc.transform.MegaPhase.goApply(MegaPhase.scala:607)
	at dotty.tools.dotc.transform.MegaPhase.transformUnnamed$1(MegaPhase.scala:272)
	at dotty.tools.dotc.transform.MegaPhase.transformTree(MegaPhase.scala:392)
	at dotty.tools.dotc.transform.MegaPhase.mapDefDef$1(MegaPhase.scala:245)
	at dotty.tools.dotc.transform.MegaPhase.transformNamed$1(MegaPhase.scala:248)
	at dotty.tools.dotc.transform.MegaPhase.transformTree(MegaPhase.scala:391)
	at dotty.tools.dotc.transform.MegaPhase.transformStat$1(MegaPhase.scala:400)
	at dotty.tools.dotc.transform.MegaPhase.$anonfun$transformStats$2(MegaPhase.scala:405)
	at scala.collection.immutable.List.mapConserve(List.scala:175)
	at dotty.tools.dotc.transform.MegaPhase.transformStats(MegaPhase.scala:405)
	at dotty.tools.dotc.transform.MegaPhase.transformUnnamed$1(MegaPhase.scala:339)
	at dotty.tools.dotc.transform.MegaPhase.transformTree(MegaPhase.scala:392)
	at dotty.tools.dotc.transform.MegaPhase.transformNamed$1(MegaPhase.scala:251)
	at dotty.tools.dotc.transform.MegaPhase.transformTree(MegaPhase.scala:391)
	at dotty.tools.dotc.transform.MegaPhase.transformStat$1(MegaPhase.scala:400)
	at dotty.tools.dotc.transform.MegaPhase.$anonfun$transformStats$2(MegaPhase.scala:405)
	at scala.collection.immutable.List.mapConserve(List.scala:175)
	at dotty.tools.dotc.transform.MegaPhase.transformStats(MegaPhase.scala:405)
	at dotty.tools.dotc.transform.MegaPhase.mapPackage$1(MegaPhase.scala:356)
	at dotty.tools.dotc.transform.MegaPhase.transformUnnamed$1(MegaPhase.scala:359)
	at dotty.tools.dotc.transform.MegaPhase.transformTree(MegaPhase.scala:392)
	at dotty.tools.dotc.transform.MegaPhase.transformUnit(MegaPhase.scala:411)
	at dotty.tools.dotc.transform.MegaPhase.run(MegaPhase.scala:423)
	at dotty.tools.dotc.core.Phases$Phase.$anonfun$runOn$1(Phases.scala:297)
	at scala.collection.immutable.List.map(List.scala:282)
	at dotty.tools.dotc.core.Phases$Phase.runOn(Phases.scala:295)
	at dotty.tools.dotc.core.Phases$Phase.runOn$(Phases.scala:294)
	at dotty.tools.dotc.transform.MegaPhase.runOn(MegaPhase.scala:137)
	at dotty.tools.dotc.Run.$anonfun$compileUnits$3(Run.scala:172)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at dotty.tools.dotc.util.Stats$.trackTime(Stats.scala:49)
	at dotty.tools.dotc.Run.$anonfun$compileUnits$2(Run.scala:169)
	at dotty.tools.dotc.Run.$anonfun$compileUnits$2$adapted(Run.scala:167)
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:32)
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:29)
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:194)
	at dotty.tools.dotc.Run.runPhases$1(Run.scala:167)
	at dotty.tools.dotc.Run.$anonfun$compileUnits$1(Run.scala:192)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:90)
	at dotty.tools.dotc.Run.compileUnits(Run.scala:147)
	at dotty.tools.dotc.Run.compileSources(Run.scala:134)
	at dotty.tools.dotc.Run.compile(Run.scala:118)
	at dotty.tools.dotc.Driver.doCompile(Driver.scala:30)
	at dotty.tools.dotc.Driver.process(Driver.scala:136)
	at dotty.tools.dotc.Driver.process(Driver.scala:105)
	at dotty.tools.dotc.Driver.process(Driver.scala:117)
	at dotty.tools.dotc.Driver.main(Driver.scala:144)
	at dotty.tools.dotc.Main.main(Main.scala)
```

--- a/compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala
+++ b/compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala
@@ -13,6 +13,7 @@ import Constants._
 import Decorators._
 import Denotations._, SymDenotations._
 import dotty.tools.dotc.ast.tpd
+import TypeErasure.erasure
 import DenotTransformers._
 
 object ElimRepeated {
@@ -92,7 +93,7 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
       JavaSeqLiteral(elems, elemtpt)
     case _ =>
       val elemType = tree.tpe.elemType
-      var elemClass = elemType.classSymbol
+      var elemClass = erasure(elemType).classSymbol
       if (defn.NotRuntimeClasses.contains(elemClass)) elemClass = defn.ObjectClass
       ref(defn.DottyArraysModule)
         .select(nme.seqToArray)

--- a/tests/pos/i5411.scala
+++ b/tests/pos/i5411.scala
@@ -1,0 +1,5 @@
+trait A
+trait B
+object O {
+  def m(x: Seq[A & B]) = java.util.Arrays.asList(x: _*)
+}

